client/my-sites/stats/site.jsx

```diff
@@ -300,28 +288,36 @@ class StatsSite extends Component {
 							/>
 						</div>
 					</div>
-					{ ! isVip && isAdmin && ! hasWordAds && (
-						<Banner
-							className="stats__upsell-nudge"
-							icon="star"
-							title={ translate( 'Start earning money now' ) }
-							description={ translate(
-								'Accept payments for just about anything and turn your website into a reliable source of income with payments and ads.'
+				</div>
+				<Card className="stats__parsely-banner">
+					<TrackComponentView eventName="calypso_stats_parsely_banner_view" />
+					<img src={ parselyIcon } alt="" />
```

Comment:
Maybe give this `aria-hidden="true"` since it's only aesthetic and doesn't have an alt?